soliket/ccl.py

```diff
@@ -175,3 +200,61 @@ def get_CCL(self):
         :return: dict of results
         """
         return self._current_state['CCL']
+
+class Tester(Likelihood):
```

Comment:
If this is something that should stay here in `ccl.py`, can it have a more informative name?  Is this a general purpose cross-correlation likelihood?  Maybe give it a more informative name and a bit of a doc string to explain what it is?